### PR TITLE
WIP: Update README.md to define non-members contribution policies

### PR DESCRIPTION
**Pull request by mmccool**
We should define the policy for accepting contributions from non-member.

### CODE REVIEW

**Comment by mmccool** [comment]
@wseltzer can you please review and let us know how we should handle contributions from "guests"?   Actually, we also need to deal with this in the wot-usecases repository.   This PR states a *proposed* policy.

**Comment by mmccool** [comment]
A template for a "statement of agreement" might be a good idea.  Contributions can include a copy with names filled in.  The following is used for the TD repo but that is a WG repo, so not directly applicable, since this is an IG repo:
* https://github.com/w3c/wot-thing-description/blob/master/CONTRIBUTING.md
* https://github.com/w3c/wot-thing-description/blob/master/LICENSE.md

**Comment by mmccool** [comment]
Perhaps we can configure the repo so that a notice shows up automatically when a PR is submitted.

File: proposals/README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-empty
+# Proposals
+This directory may include contributions from non-IG members.
+Contributors agree to abide by the terms of the W3C Patent and document license policies:
+* [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/)
+    - [Patent Policy FAQ - Q6](https://www.w3.org/2003/12/22-pp-faq.html#non-participants)
+* [W3C Document License Policy](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document)
```
Comment (xfq) [inline proposals/README.md:6]: IANAL but I wonder if it should be W3C Software and Document Notice and License? [W3C Document License](https://www.w3.org/Consortium/Legal/2015/doc-license) and [W3C Software and Document Notice and License](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document) are different licenses.

**Comment by sebastiankb** [comment]
from today's marketing call:
- keep this PR open
- we have to check if we should include the W3C Software and Document Notice and License (also see comment [here](https://github.com/w3c/wot-marketing/pull/89#discussion_r543426529)). So far, marketing TF do not develop software, we have only scripts in our webpage
- we need to check with @koalie  and @plehegar

**Comment by koalie** [comment]
> from today's marketing call:
> 
>     * keep this PR open
[...]
>     * we need to check with @koalie  and @plehegar

I defer to @plehegar (I have no expertise in what groups ought to state with regards to non-member contributors.)

**Comment by danielpeintner** [comment]
@mmccool will update license to match IG charter license.

PRs content should agree to given license.

**Comment by mmccool** [comment]
* Updated to reference W3C Software and Document license
* Slight reorg (use "license" rather than "license policy", make clear contributors must agree to these terms)

**Comment by mmccool** [comment]
So one question is what the repo IPR check is.  It may not allow actual contributions from non-members, in which case this is moot.

**Comment by mmccool** [comment]
So... technically the W3C Document and Software link (https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document) is for a *Notice* and a License, and the notice needs to go somewhere (e.g. in comments).   So...
- Small contributions (e.g. adding a link to an existing document) just need to agree to an existing notice
- We probably need to add notices to our *existing* content
- Entirely content should include the notice, e.g. in a header; maybe provide a template?
- I *assume* putting the notice in an (invisible) comment is ok, although maybe technically we also need to link it from the footer or something for HTML content
- Software for tutorials etc. should probably include the notice in a comment header

**Comment by mmccool** [comment]
Reading this more carefully:
- There is a "short" notice that can be used if there are no other rights: https://www.w3.org/Consortium/Legal/2015/copyright-software-short-notice.html
- Statement in license says "The full text of this NOTICE in a location viewable to users of the redistributed or derivative work."; but I *assume* (need to ask for clarification) for HTML this can still be in a comment (viewable if you look at the document source) and/or in a link in the footer (viewable if the user follows the link).

**Comment by FadySalama** [comment]
TBD if needed

**Pull request closed without merging**